src/pages/library/DataBrowser.js

```diff
@@ -221,11 +225,30 @@ const Browser = () => {
   const [selectedData, setSelectedData] = useState([])
   const [requestDatasetAccessList, setRequestDatasetAccessList] = useState()
   const { dataCatalog, loading } = useDataCatalog()
+  const acknowledged = Utils.useStore(acknowledgmentStore)
```

Comment:
This will give us the user ID when it populates
```suggestion
  const acknowledged = Utils.useStore(acknowledgmentStore) || {}
  const { user: { id } } = Utils.useStore(authStore)
```